controllers/userController.js

```diff
@@ -0,0 +1,43 @@
+const User = require('../model/userModel')
+const bcrypt = require('bcrypt')
+const jwt = require('jsonwebtoken')
+
+const register = async (req, res) => {
+  try {
+    const { firstName, lastName, email, password } = req.body
+
+    const emailLowerCase = email.toLowerCase()
+
+    const existingUser = await User.findOne({ email: emailLowerCase })
+
+    if (existingUser) {
+      return res.status(409).json({ error: 'User Already Exist. Please Login' })
+    }
+
```

Comment:
You don't need to check that in code, it's must be a constraint on database configuration